src/main/java/org/takes/facets/auth/PsBasic.java

```diff
@@ -141,7 +150,7 @@ public Response exit(final Response response, final Identity identity)
      * <p>The class is immutable and thread-safe.
      *
      * @author Endrigo Antonini (teamed@endrigo.com.br)
-     * @version $Id$
+     * @version $Id: 5913e70e4877ecdd3f0cb85821dd8593f48a166a $
```

Comment:
@dalifreire remove the unneeded string at the end, this field should be `@version $Id$` only